src/baldr/CMakeLists.txt

```diff
@@ -0,0 +1,110 @@
+add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/date_time_zonespec.h
+  COMMAND ${CMAKE_COMMAND} -P ${VALHALLA_SOURCE_DIR}/cmake/Binary2Header.cmake
+      ${VALHALLA_SOURCE_DIR}/date_time/zonespec.csv
+      ${CMAKE_CURRENT_BINARY_DIR}/date_time_zonespec.h
+      --variable-name date_time_zonespec_csv --skip 1 --raw
+  WORKING_DIRECTORY ${VALHALLA_SOURCE_DIR}
+  COMMENT "Compiling date_time/zonespec.csv to date_time_zonespec.h"
+  DEPENDS ${VALHALLA_SOURCE_DIR}/date_time/zonespec.csv)
+
+set(headers
```

Comment:
Could we also not hard-code these paths? It seems error prone to me since we could easily forget to install headers.